modules/ebi-metagenomics/cat/environment.yml

```diff
@@ -0,0 +1,8 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+name: "cat"
```

Comment:
name is not used anymore, linting with fail with it

Reply:
it doesn't fail, the schema says that the name is not required